simulation_parameters/Constants.cs

```diff
@@ -381,7 +388,7 @@ public static class Constants
     public const float CELL_REQUIRED_DRAG_BEFORE_APPLY = 0.0033f;
 
     public const float CHEMORECEPTOR_RANGE_MIN = 2;
-    public const float CHEMORECEPTOR_RANGE_MAX = 700;
+    public const float CHEMORECEPTOR_RANGE_MAX = CLOUD_SPAWN_RADIUS;
```

Comment:
If we assume that cloud spawn isn't equal to the compound cloud plane distance, restricting this distance doesn't seem like the best idea to me...

Reply:
This really was an attempt to fix a related but unnecessary issue anyway.